cfme/infrastructure/host.py

```diff
@@ -327,7 +327,8 @@ def get_datastores(self):
         datastores = set([])
         for page in paginator.pages():
             for title in sel.elements(
-                    "//div[@id='quadicon']/../../../tr/td/a[contains(@href,'storage/show')]"):
+                    "//div[contains(@id, 'quadicon')]/../../../tr/td/"
+                    "a[contains(@href,'storage/show')]"):
```

Comment:
I think it will be better to convert this to Quadicon.all() and then wait for upstream to be fixed (the title).

Some down there.